d3a_api_client/websocket_device.py

```diff
@@ -30,6 +30,9 @@ def received_message(self, message):
         elif "command" in message:
             self.command_response_buffer.append(message)
 
+        if "event" in message.keys() or "command" in message.keys():
+            self.client.on_event_or_response(message)
+
```

Comment:
`Rest` Devices are receiving new messages through websockets, this function is forwarding them, so additionally it will forward the message once again to the `on_event_or_response`